src/job_svc/mod.rs

```diff
@@ -0,0 +1,61 @@
+pub mod error;
+mod populate_outbox;
+
+use job_crate::{JobId, JobSvcConfig, Jobs};
+use tracing::instrument;
+
+use crate::job_svc::populate_outbox::{PopulateOutboxJobConfig, PopulateOutboxJobInit};
+use crate::{
+    ledger::Ledger,
+    outbox::Outbox,
+    primitives::{AccountId, LedgerJournalId},
+};
+
+pub use error::JobSvcError;
+
+#[derive(Clone)]
+pub struct JobSvc {
+    jobs: Jobs,
+}
+
+impl JobSvc {
+    pub async fn init(
+        pool: sqlx::PgPool,
+        outbox: Outbox,
+        ledger: Ledger,
+    ) -> Result<Self, JobSvcError> {
+        let job_svc_config = JobSvcConfig::builder()
+            .pool(pool)
+            .build()
+            .expect("Couldn't build JobSvcConfig");
+
+        let mut jobs = Jobs::init(job_svc_config).await?;
+        jobs.add_initializer(PopulateOutboxJobInit::new(outbox, ledger));
+        jobs.start_poll().await?;
+
+        Ok(Self { jobs })
+    }
+
+    pub fn jobs(&self) -> &Jobs {
+        &self.jobs
+    }
+
+    #[instrument(name = "job_svc.spawn_outbox_handler_in_op", skip_all)]
+    pub async fn spawn_outbox_handler_in_op(
+        &self,
+        op: &mut impl es_entity::AtomicOperation,
+        account_id: AccountId,
+        journal_id: LedgerJournalId,
+    ) -> Result<(), JobSvcError> {
+        let config = PopulateOutboxJobConfig {
+            account_id,
+            journal_id,
+            tracing_data: crate::tracing::extract_tracing_data(),
```

Comment:
I think this shouldn't be necessary anymore as the job crate has support for tracing OOTB - can you check on honeycomb?

Reply:
I tested after removing this field and the traces still appeared on honeycomb, so have removed this.